code/modules/projectiles/guns/ego_gun/waw.dm

```diff
@@ -293,9 +296,10 @@
 	force = 35
 	damtype = WHITE_DAMAGE
 	armortype = WHITE_DAMAGE
-	fire_delay = 24
+	fire_delay = 16
```

Comment:
Another buff..?

Reply:
This one is different; weapon is unreleased as of now. Me touching this variable has no impact on gameplay